docs/resources/metadata.md

```diff
@@ -0,0 +1,61 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "coder_metadata Resource - terraform-provider-coder"
+subcategory: ""
+description: |-
+  Use this resource to attach key/value pairs to a resource. They will be displayed in the Coder dashboard.
+---
+
+# coder_metadata (Resource)
+
+Use this resource to attach key/value pairs to a resource. They will be displayed in the Coder dashboard.
+
+## Example Usage
+
+```terraform
+data "coder_workspace" "me" {
+}
+
+resource "kubernetes_pod" "dev" {
+  count = data.coder_workspace.me.start_count
+}
+
+resource "coder_metadata" "pod_info" {
+  count = data.coder_workspace.me.start_count
+  resource_id = kubernetes_pod.dev[0].id
+  pair {
+    key = "pod_uid"
+    value = kubernetes_pod.dev[0].uid
+  }
```

Comment:
I was confused about why `key` and `value` were nested until @kylecarbs explained to me that HCL allows multiple pair blocks. We should demonstrate that here.

Reply:
Good point! I updated the example to use multiple blocks, and also to demonstrate the `sensitive` attribute.